src/olympia/api/permissions.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, app, action):
         self.action = action
 
     def has_permission(self, request, view):
-        return acl.action_allowed(request, self.app, self.action)
+        return acl.action_allowed_user(request.user, self.app, self.action)
```

Comment:
I guess this isn't used anywhere so it shouldn't cause any problems. Do anonymous users have groups? Maybe it should check `request.user.is_authenticated()`?